### ISSUE TITLE
DL4J: UNet pretrained model configuration issue

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/deeplearning4j#6955]
Has sigmoid going into softmax... this is almost certainly wrong, unless there's some unusual reason for doing this in UNet...

https://github.com/deeplearning4j/deeplearning4j/blob/master/deeplearning4j/deeplearning4j-zoo/src/main/java/org/deeplearning4j/zoo/model/UNet.java#L219-L223

**Comment by emildb** [comment]
I think the following changes should be made to the model: Remove the last conv layer ("conv10") which outputs 1 feature map (this layer does not exist in the original paper) and change the activation function in the new last conv layer ("conv9-3") to identity with two output feature maps (which is the same as in the original paper) (the number of output feature maps will correspond to the number of segmentation classes).
